binary-compatibility-validator/reference-public-api/kotlinx-coroutines-core.txt

```diff
@@ -1070,13 +1072,18 @@ public abstract interface class kotlinx/coroutines/selects/SelectInstance {
 	public abstract fun isSelected ()Z
 	public abstract fun performAtomicTrySelect (Lkotlinx/coroutines/internal/AtomicDesc;)Ljava/lang/Object;
 	public abstract fun resumeSelectCancellableWithException (Ljava/lang/Throwable;)V
-	public abstract fun trySelect (Ljava/lang/Object;)Z
+	public abstract fun trySelect ()Z
+	public abstract fun trySelectOther (Lkotlinx/coroutines/internal/LockFreeLinkedListNode$PrepareOp;)Ljava/lang/Object;
 }
 
 public final class kotlinx/coroutines/selects/SelectKt {
 	public static final fun select (Lkotlin/jvm/functions/Function1;Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
 }
 
+public synthetic class kotlinx/coroutines/selects/SelectKtSelectOpSequenceNumberRefVolatile {
```

Comment:
Unfortunately, this is how `Atomicfu` is currently implemented. When faced with a static atomic field it creates a separate container class using this simple naming scheme and makes it public. Here's the issue: https://github.com/Kotlin/kotlinx.atomicfu/issues/84 

In fact, it would be better if it did not create a class at all, but used the corresponding `j.u.c.AtomicXxx` class from Java, which would require more work in `Atomicfu` implementation to properly map the methods to the corresponding `j.u.c.Atomic.Xxx` methods. As a workaround, I'll wrap it into an separate property named class.